Formula/python.rb

```diff
@@ -176,79 +154,82 @@ def install
 
     system "./configure", *args
 
-    # HAVE_POLL is "broken" on macOS. See:
-    # https://trac.macports.org/ticket/18376
-    # https://bugs.python.org/issue5154
-    if build.without? "poll"
-      inreplace "pyconfig.h", /.*?(HAVE_POLL[_A-Z]*).*/, '#undef \1'
-    end
-
     system "make"
-    if build.with?("quicktest") || build.bottle?
+    if build.with?("quicktest")
       system "make", "quicktest", "TESTPYTHONOPTS=-s", "TESTOPTS=-j#{ENV.make_jobs} -w"
     end
 
     ENV.deparallelize do
-      # Tell Python not to install into /Applications
+      # Tell Python not to install into /Applications (default for framework builds)
       system "make", "install", "PYTHONAPPSDIR=#{prefix}"
       system "make", "frameworkinstallextras", "PYTHONAPPSDIR=#{pkgshare}"
     end
 
-    # Fixes setting Python build flags for certain software
-    # See: https://github.com/Homebrew/homebrew/pull/20182
-    # https://bugs.python.org/issue3588
-    inreplace lib_cellar/"config/Makefile" do |s|
-      s.change_make_var! "LINKFORSHARED",
-        "-u _PyMac_Error $(PYTHONFRAMEWORKINSTALLDIR)/Versions/$(VERSION)/$(PYTHONFRAMEWORK)"
-    end
+    # Any .app get a " 3" attached, so it does not conflict with python 2.x.
```

Comment:
Isn't `linkapps` pretty much dead now, rendering anything regarding `app` versioning redundant? If not, should possibly be the other way around, with the new `keg_only` `python@2` formula getting the versioned apps and this formula the standard names?